types/chrome/index.d.ts

```diff
@@ -7616,6 +7616,25 @@ declare namespace chrome.scripting {
 
     type Awaited<T> = T extends PromiseLike<infer U> ? U : T;
 
+    type RegisteredContentScript = {
+        allFrames?: boolean;
+        css?: string[];
+        excludeMatches?: string[];
+        id: string;
+        js?: string[];
+        matches?: string[];
+        persistAcrossSessions?: boolean;
+        runAt?: "document_start" | "document_end" | "document_idle";
+        world?: ExecutionWorld;
+    };
+
+    type ContentScriptFilter = {
+        ids?: string[];
+        css?: string;
+        files?: string[];
+        origin?: StyleOrigin;
+        target?: InjectionTarget;
+
```

Comment:
These would be better as interfaces instead of type literals (and the last one needs a closing curly brace). This is currently failing CI.